src/Suite.ts

```diff
@@ -22,6 +22,9 @@ async function writeTests() {
     });
 
     each(storiesByComponent, (stories, componentTitle) => {
+      if (!componentTitle.match(options.pattern)) {
```

Comment:
another option is to mark the test file as skipped

Reply:
That's a good idea! What do you think about skipping the `describe` when the component matches?